src/about.cpp

```diff
@@ -106,11 +101,15 @@ std::optional<credits_data::const_iterator> get_campaign_credits(const std::stri
 
 std::vector<std::string> get_background_images(const std::string& campaign)
 {
-	if(!campaign.empty() && !images_campaigns[campaign].empty()) {
-		return images_campaigns[campaign];
+	if(campaign.empty()) {
+		return images_general;
 	}
 
-	return images_general;
+	if(const auto it = images_campaigns.find(campaign); it == images_campaigns.cend()) {
```

Comment:
I feel that `if(...; it != images_campaign.cend())` would be better - put the handling for there being images in the `if` block, and the fallback when there aren't images in the `else` block.

Generally, I find the old version of this function easier to read.

Reply:
I'll rework it to look similar to that one while still avoiding the double operator[]

Reply:
Made the change